putty.sls

```diff
@@ -1,11 +1,27 @@
 # just 32-bit x86 installer available
-{% if grains['cpuarch'] == 'AMD64' %}
+putty:
+{% for version in '0.68' %}
```

Comment:
Does `'0.68'` need to be in brackets? I think this may just iterate over the contents of the '0.68' string.